runtime/tests/mock.rs

```diff
@@ -94,6 +98,10 @@ impl ExtBuilder {
 		self.validator_count = count;
 		self
 	}
+	pub fn sudoer(mut self, key: AccountId32) -> Self {
```

Comment:
Doubt here, do we need changes for scaling in tests/mock.rs too? As we now have separate module and its mock contains all these related function..

Reply:
Yup, you are right. we don't need this anymore here. Thank you for seeing it.